code-samples/eventing/bookstore-sample-app/db/sample.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE BookReviews(
+  id SERIAL PRIMARY KEY,
+  post_time datetime NOT NULL,
+  content TEXT NOT NULL,
+  sentiment TEXT,
+  CONSTRAINT sentiment_check CHECK (sentiment IN ('positive', 'negative', 'neutral')),
+);
+
+INSERT INTO BookReviews VALUES(1, '2020-01-01 00:00:00', 'This book is great!', 'positive');
+INSERT INTO BookReviews VALUES(2, '2020-01-02 00:02:00', 'This book is terrible!', 'negative');
+INSERT INTO BookReviews VALUES(3, '2020-01-03 00:01:30', 'This book is okay.', 'neutral');
+INSERT INTO BookReviews VALUES(4, '2020-01-04 00:00:00', 'Meh', 'neutral');
```

Comment:
This is probably better as a single set of inserts, and also when you have a `SERIAL` type in postgres it is best to let postgres increment that field rather than doing it yourself otherwise you can run into scenarios where postgres doesn't know about the newest value you have
```suggestion
INSERT INTO book_reviews (post_time, content, sentiment) VALUES 
    ('2020-01-01 00:00:00', 'This book is great!', 'positive'),
    ('2020-01-02 00:02:00', 'This book is terrible!', 'negative'),
    ('2020-01-03 00:01:30', 'This book is okay.', 'neutral'),
    ('2020-01-04 00:00:00', 'Meh', 'neutral');
```